Sources/PointFree/LaunchSignup/Views.swift

```diff
@@ -153,18 +153,15 @@ let responsiveSuccessStyle =
     <> bottom(0)
 
 let responsiveStyle = queryOnly(screen, [minWidth(.px(800))]) {
-  ".hero, .signup, .success" % (
```

Comment:
Another 9ish second bit of code down to under 500 ms.

Reply:
yeeeesh